github/table_github_audit_log.go

```diff
@@ -0,0 +1,146 @@
+package github
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/go-github/v45/github"
+	pb "github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableGitHubAuditLog(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "github_audit_log",
+		Description: "Gets the audit log for an organization.",
+		List: &plugin.ListConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "organization", Require: plugin.Required},
+				{Name: "phrase", Require: plugin.Optional},
+				{Name: "include", Require: plugin.Optional},
+				{Name: "action", Require: plugin.Optional},
+				{Name: "actor", Require: plugin.Optional},
+				{Name: "created_at", Require: plugin.Optional, Operators: []string{">", ">=", "<", "<=", "="}},
+			},
+			Hydrate: tableGitHubAuditLogList,
+		},
+		Columns: []*plugin.Column{
```

Comment:
We don't necessarily need (or even want to include) all of the properties as columns for an audit log entry, but looking at https://github.com/google/go-github/blob/fd22ee93382e50d36592170f63c6a08d7c35ed8d/github/orgs_audit_log.go#L34-L92, there appear to be a lot of columns.

Are there any key properties that you feel we're missing in this table? For instance, what about `Event` or `Events`, or `Name` and `Message`? I'm not sure exactly what's in these properties, but the number of columns here compared to the struct seem a bit light.

Reply:
@japborst that sounds good to me, let's see if there's any feedback from other enterprise users